readme.md

```diff
@@ -133,9 +133,9 @@ It'll be nice going forward with your application that we package the schema up
 
 **NOTE:** If you already ran your Tunr schema in the Domain Modeling / SQL class, you do not need to complete this portion.  
 
-[Part 1 - Database/Schema](https://github.com/ga-dc/tunr_sinatra/tree/2_active_record_starter#part-1---database--schema)
+[solution code](https://github.com/ga-wdi-exercises/tunr-active-record/archive/v1.0.zip)
```

Comment:
are you planning on having them clone down any [starter](https://github.com/ga-wdi-exercises/tunr-active-record#local-setup)?